metaflow/plugins/catch_decorator.py

```diff
@@ -84,6 +84,12 @@ def task_exception(
 
         # pretend that self.next() was called as usual
         flow._transition = (graph[step].out_funcs, None, None)
+
+        # If this task is a UBF control task, it will return itself as the singleton
+        # list of tasks.
+        flow._control_mapper_tasks = [
+            "/".join((current.run_id, current.step_name, current.task_id))
```

Comment:
is there a reason we are not also storing the full pathspec?

Reply:
Sorry I don't understand this question?

Reply:
It starts with run_id instead of flow_name. Just wondering if there is a reason it is not the full pathspec.